spec/Section 6 -- Execution.md

```diff
@@ -316,14 +316,28 @@ To execute a selection set, the object value being evaluated and the object type
 need to be known, as well as whether it must be executed serially, or may be
 executed in parallel.
 
+For the purposes of execution, the term "field" may be used to reference a
+"member" of a tagged type; this simplifies the wording of the algorithm.
+
 First, the selection set is turned into a grouped field set; then, each
 represented field in the grouped field set produces an entry into a
 response map.
 
+GetTaggedMemberName(objectType, objectValue):
+
+  * If {objectType} is not a Tagged type, return {null}.
+  * Let {keys} be the keys of {objectValue} that are member names of {objectType}.
```

Comment:
do we use "keys" terminology elsewhere?

Reply:
I don't think so. Not sure what terminology to use here?